graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndicesResource.java

```diff
@@ -173,6 +174,16 @@ public boolean apply(String indexName) {
         return ClosedIndices.create(reopenedIndices, reopenedIndices.size());
     }
 
+    @GET
+    @Timed
+    @Path("/all")
+    @ApiOperation(value = "Get a list of all indices, open, closed and reopened.")
+    @Produces(MediaType.APPLICATION_JSON)
+    public BulkIndices bulk() {
```

Comment:
I think the whole API needs a bigger overhaul. The naming is inconsistent all over the place. I would prefer to decouple this from the creation of a bulk call though. Maybe you can spend some time doing that outside of the scope of this PR?